### ISSUE TITLE
Conditional questions causing plans to disapear 

### ISSUE THREAD

**Issue by mariapraetzellis** [DMPRoadmap/roadmap#3163]
Plans that include a conditional question are disappearing. For example, a user can successfully create a plan from a template including a conditional question, but when they go back to view the plan, the plan will disappear from the page. 

We temporarily fixed this issue by removing the conditional question from the template, but I believe this bug will be affecting all plans that include conditional questions.

**Comment by johnpinto1** [comment]
@raycarrick-ed This seems to be related to issue I noticed when dealing with a bug for Maastricht. The plan sections container continues to have style="display:none". Will investigate.

**Comment by johnpinto1** [comment]
@mariapraetzellis, @briri Not yet identified cause, but working on it. As I have an example which fails.

**Comment by johnpinto1** [comment]
Just proved in broken case cause this rendering cuases everything in **content_for** to be surrouned by a display:none  block. 
  ```
<%= render partial: 'phases/edit_plan_answers',
               locals: {
                 plan: plan,
                 phase: phase,
                 answers: answers,
                 readonly: readonly,
                 base_template_org: base_template_org,
                 guidance_presenter: guidance_presenter,
               } %>
```

**Comment by johnpinto1** [comment]
Thanks @briri. A cursory test shows commenting out hide questions fixes issue.

Will tomorrow dig deeper as suggested in your comment.

**Comment by pherterich** [comment]
I think @raycarrick-ed added this to the latest release, so I will close out here.

**Comment by johnpinto1** [comment]
@pherterich I can't see it in https://github.com/DigitalCurationCentre/roadmap/commits/deploy/dmponline?before=99d780716fca202f19a6dcb7da40ce62edb52e76+35&branch=deploy%2Fdmponline&qualified_name=refs%2Fheads%2Fdeploy%2Fdmponline 
It is possible has been patched by @raycarrick

**Comment by pherterich** [comment]
It is patched and part of release 3.1.1 proper
